subprojects/core/src/integTest/groovy/org/gradle/api/tasks/TaskPropertiesIntegrationTest.groovy

```diff
@@ -101,8 +100,121 @@ class TaskPropertiesIntegrationTest extends AbstractIntegrationSpec {
         failure.assertHasCause("Cannot query the value of task ':thing' property 'count' because it has no value available.")
     }
 
-    @ToBeFixedForConfigurationCache(skip = INVESTIGATE)
-    def "reports failure to query read-only unmanaged Property<T>"() {
+    @Issue("https://github.com/gradle/gradle/issues/33215")
+    def "non-abstract Property<T> with Groovy property carries task dependencies"() {
+        given:
+        buildFile << """
+            abstract class MyTask extends DefaultTask {
+                @OutputFile
+                final Property<String> output = project.objects.property(String)
```

Comment:
Does this need to be a `Property<String>`?  Could it just be a `RegularFileProperty`?  I'm not sure what an `@OutputFile` on a `Property<String>` means.